macrodata.js

```diff
@@ -0,0 +1,59 @@
+// TODO: add more file names
+const files = [
+  'Siena',
+  'Nanning',
+  'Narva',
+  'Ocula',
+  'Kingsport',
+  'Labrador',
+  'Le Mars',
+  'Longbranch',
+  'Moonbeam',
+  'Minsk',
+  'Dranesville',
+];
+
+const macrodataKey = 'macrodata';
+
+const emptyBins = [
+  {WO: 0, FC: 0, DR: 0, MA: 0},
+  {WO: 2, FC: 0, DR: 0, MA: 0},
+  {WO: 0, FC: 0, DR: 0, MA: 0},
+  {WO: 0, FC: 0, DR: 0, MA: 0},
+  {WO: 0, FC: 0, DR: 0, MA: 0}
+ ];
+
+// would it be better to wrap this functionality in a class?
+
+const fetchFile = (filename) => {
```

Comment:
I'm used to using JS Modules to avoid adding things to global scope. Would it be better to wrap these methods in a MacrodataFile class to avoid adding so many things to global scope?